packages/core/components/EditMetadata/NewAnnotationPathway.tsx

```diff
@@ -18,21 +31,74 @@ enum EditStep {
 
 interface NewAnnotationProps {
     onDismiss: () => void;
-    onUnsavedChanges: () => void;
+    hasUnsavedChanges: (arg: boolean) => void;
     selectedFileCount: number;
 }
 
+// Simplified version of status message
+interface AnnotationStatus {
+    status: ProcessStatus;
+    message: string | undefined;
+}
+
 /**
  * Component for submitting a new annotation
  * and then entering values for the selected files
  */
 export default function NewAnnotationPathway(props: NewAnnotationProps) {
+    const dispatch = useDispatch();
+    // Destructure to prevent unnecessary useEffect triggers
+    const { onDismiss, hasUnsavedChanges } = props;
+
     const [step, setStep] = React.useState<EditStep>(EditStep.CREATE_FIELD);
-    const [newValues, setNewValues] = React.useState<string | undefined>();
+    const [newValues, setNewValues] = React.useState<AnnotationValue | undefined>();
     const [newFieldName, setNewFieldName] = React.useState<string>("");
+    const [newFieldDescription, setNewFieldDescription] = React.useState<string>("");
     const [newFieldDataType, setNewFieldDataType] = React.useState<AnnotationType | undefined>();
     const [newDropdownOption, setNewDropdownOption] = React.useState<string>("");
     const [dropdownOptions, setDropdownOptions] = React.useState<IComboBoxOption[]>([]);
+    const [submissionStatus, setSubmissionStatus] = React.useState<AnnotationStatus | undefined>();
+
+    const statuses = useSelector(interaction.selectors.getProcessStatuses);
+    const annotationCreationStatus = React.useMemo(
+        () => statuses.find((status) => status.processId === newFieldName),
+        [newFieldName, statuses]
+    );
+    // Check for updates to the annotation submission status
+    React.useEffect(() => {
```

Comment:
We aren't able to directly chain off of the createAnnotation dispatch (we don't use thunk in this codebase) so instead we need to wait for the status to update in state and then respond accordingly when we see `SUCCEEDED`